haml2slim.gemspec

```diff
@@ -22,4 +22,6 @@ Gem::Specification.new do |s|
   s.add_dependency "ruby_parser"
   s.add_development_dependency 'rake'
   s.add_development_dependency "slim", [">= 1.0.0"]
+  s.add_development_dependency 'minitest', '~> 5.8.3'
+  s.add_development_dependency 'pry'
```

Comment:
you added this as a development dependency, but it appears required at runtime as well.